locals.tf

```diff
@@ -254,15 +254,16 @@ locals {
   hive_metastore_location = "data/dataworks-aws-corporate-data-ingestion"
 
   run_daily_export_on_schedule = {
-    development = false
+    development = true
```

Comment:
Do we want dev to run once a day?

Reply:
Initially at least I think that would be nice